tests/test_repository_tool.py

```diff
@@ -1208,10 +1208,10 @@ def test_delegate(self):
                       path_hash_prefixes=path_hash_prefixes)
 
     # Test for targets that do not exist under the targets directory.
-    self.assertRaises(securesystemslib.exceptions.Error, self.targets_object.delegate, rolename,
-                      public_keys, list_of_targets, threshold,
-                      terminating=False, restricted_paths=['non-existent.txt'],
-                      path_hash_prefixes=path_hash_prefixes)
+    # An exception should be raised for non-existent delegated paths.
```

Comment:
This comment says that an exception should be raised, but the the test call does not assert for an exception. Maybe the comment should say why this call does not raise an exception.